### ISSUE TITLE
Geosearch keyboard navigation

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#3784]
When I use the keyboard to with Geosearch, navigation get stuck inside the panel. If you enter e.g. Toronto inside the text field then tab to the result to enter the result panel, there is no way to get out. The esc key that use to work in 2.5 doesn't work anymore. At the same time, tab order is not good compare to 2.5.

```
angular.js:15536 TypeError: o.closePanelFromHistory is not a function
    at shell.directive.js:76
    at u.$eval (angular.js:19356)
    at u.$apply (angular.js:19455)
    at HTMLDivElement.<anonymous> (shell.directive.js:75)
    at HTMLDivElement.dispatch (event.js:353)
    at HTMLDivElement.b.handle (event.js:160)
(anonymous) @ angular.js:15536
(anonymous) @ angular.js:11815
$apply @ angular.js:19460
(anonymous) @ shell.directive.js:75
dispatch @ event.js:353
b.handle @ event.js:160
```

**Comment by YaroRoodzit** [comment]
Tested build: v3.3.0-b2
Keyboard navigation ( tabbing/esc) works correctly in Geosearch results.
Issue solved.
